browser/css/device-desktop.css

```diff
@@ -9,7 +9,7 @@
 	margin-left: 1px !important;
 	transform: translateY(-12px);
 }
-#tb_formulabar_item_formula .inputbar_selection_handles .leaflet-selection-marker-start{
+#tb_formulabar_item_formula .inputbar_selection_handles .text-selection-handle-start{
```

Comment:
`#tb_formulabar_item_formula` doesn't exists anymore after w2ui removing.
We can drop this rule then